openapi.yaml

```diff
@@ -953,7 +953,7 @@ paths:
         - customers
       summary: Delete a customer
       description: Return the deleted customer
-      operationId: DeleteCustomer
+      operationId: deleteCustomer
```

Comment:
Other delete operations are called `destroy*`, so maybe we could name this one `destroyCustomer`